src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,26 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(1, $card-width);
```

Comment:
Ensure that the variable `$card-width` is defined in your SCSS files. If not, define it to avoid compilation errors.